nncf/quantization/algorithms/min_max/algorithm.py

```diff
@@ -382,6 +382,25 @@ def _get_ignored_scope_by_algorithm(self, inference_nncf_graph: NNCFGraph) -> Ig
                 nncf_node_names.append(nncf_node.node_name)
         return IgnoredScope(names=nncf_node_names)
 
+    def _get_scope_overrides(self, inference_nncf_graph: NNCFGraph) -> Dict:
+        """
+        Returns a dictionary of quantization configuration overrides for inputs to matching operation nodes.
+
+        :param inference_nncf_graph: Inference NNCFGraph instance.
+        :return: A dictionary of quantization configuration overrides for inputs to matching operation nodes.
+        """
+        scaled_dot_product_attention_node_names = [
+            node.node_name
+            for node in inference_nncf_graph.get_nodes_by_metatypes(
+                self._backend_entity.scaled_dot_product_attention_metatypes
+            )
+        ]
+
+        scope_overrides_activations = {}
+        for node_name in scaled_dot_product_attention_node_names:
+            scope_overrides_activations[node_name] = {"mode": "symmetric"}
```

Comment:
Why do we need this override since we have only one possible `q8_a_sym` configuration for these layers?

Reply:
Yes, you are absolutely right.

Reply:
Still no answer, why do we need this?